integration/airflow/openlineage/airflow/extractors/manager.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Optional
+
+from openlineage.airflow.extractors import TaskMetadata, BaseExtractor, Extractors
+from openlineage.airflow.utils import get_job_name
+
+
+class ExtractorManager:
+    """Class abstracting management of custom extractors."""
+    def __init__(self):
+        self.extractors = {}
+        self.extractor_mapper = Extractors()
```

Comment:
_Minor_: To be a bit more clear on the mapping, I'd use `task_to_extractor` instead of `extractor_mapper`